packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -4,7 +4,79 @@ import type { Settings } from './generated-types'
 import forwardProfile from './forwardProfile'
 import forwardAudienceEvent from './forwardAudienceEvent'
 import { AdvertiserScopesResponse } from './types'
-import { GQL_ENDPOINT } from './functions'
+import { GQL_ENDPOINT, EXTERNAL_PROVIDER } from './functions'
```

Comment:
Let's move this block for onDelete handling to its own folder. You can read about modularization: https://medium.com/@caitlinjeespn/modularization-in-software-engineering-1af52807ceed